library/src/scala/compiletime/testing/package.scala

```diff
@@ -4,6 +4,15 @@ import scala.quoted._
 
 package object testing {
 
+  /** Represents a compile-time error */
```

Comment:
Specify in the scaladoc that this interface is unstable from version to version.